src/components/describe/describeWebview.ts

```diff
@@ -28,11 +56,51 @@ export class DescribePanel extends WebPanel {
     <head>
         <meta charset="UTF-8">
         <title>Kubernetes describe ${this.resource}</title>
+        <script>
+            const vscode = acquireVsCodeApi();
+            const requestRefresh = () => {
+                vscode.postMessage({
+                    command: 'refresh'
+                });
+            };
+
+            window.addEventListener('message', event => {
+                const message = event.data;
+                switch (message.command) {
+                    case 'content':
+                        const elt = document.getElementById('content');
+                        elt.innerText = message.content;
+                }
+            });
+
+            const automaticRefresh = () => {
+                const elt = document.getElementById('refresh');
+                const val = parseInt(elt.value);
+                if (val > 0) {
+                    setTimeout(() => {
+                        requestRefresh();
+                        automaticRefresh();
+                    }, val * 1000);
+                }
+            };
+        </script>
     </head>
     <body>
+        <div style="position: absolute; left: 90%">
+          <span><a href="javascript:void;" onClick="requestRefresh();">Refresh</a></span>
```

Comment:
This ends up looking a bit odd to me:

![image](https://user-images.githubusercontent.com/865538/53528272-09d5b880-3b4e-11e9-8852-05cb8bc0581f.png)

The vertical positioning seems a bit tight, and the drop-down lacks context.  Again, a toolbar might provide a bit more breathing room.

Reply:
removed.